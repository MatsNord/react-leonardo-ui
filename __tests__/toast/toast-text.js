import React from 'react';
import renderer from 'react-test-renderer';

import Toast from '../../src/toast';

test('Toast text renders correctly', () => {
  const tree = renderer.create(
    <Toast.Text>Text</Toast.Text>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Toast text passes props correctly', () => {
  const tree = renderer.create(
    <Toast.Text className="my-class" data-x="x" title="Text">
      Text
    </Toast.Text>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
