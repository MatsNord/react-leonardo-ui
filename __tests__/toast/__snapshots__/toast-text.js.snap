// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast text passes props correctly 1`] = `
<div
  className="lui-toast__text my-class"
  data-x="x"
  title="Text"
>
  Text
</div>
`;

exports[`Toast text renders correctly 1`] = `
<div
  className="lui-toast__text"
>
  Text
</div>
`;
